src/drivers/distance_sensor/lightware_sf45_serial/lightware_sf45_serial.cpp

```diff
@@ -91,16 +94,12 @@ SF45LaserSerial::~SF45LaserSerial()
 
 int SF45LaserSerial::init()
 {
-
 	param_get(param_find("SF45_UPDATE_CFG"), &_update_rate);
 	param_get(param_find("SF45_ORIENT_CFG"), &_orient_cfg);
 	param_get(param_find("SF45_YAW_CFG"), &_yaw_cfg);
 
 	/* SF45/B (50M) */
```

Comment:
We can remove this comment now that the rangefinder min/max values are no longer relevant